backend/models/events.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require("mongoose");
```

Comment:
This looks like a holdover from another branch. Could you remove the changes to this file if possible?